WcaOnRails/app/models/registration.rb

```diff
@@ -105,12 +105,15 @@ def best_solve(event, type)
   end
 
   def entry_fee
-    competition.base_entry_fee
+    Money.new(
+      competition.base_entry_fee_lowest_denomination + competition_events.sum(:event_entry_fee_lowest_denomination),
```

Comment:
Surely you can add Money objects?

I'd expect something like `competition.base_entry_fee + competition_events.sum(:event_entry_fee)` to work.